CHANGELOG.md

```diff
@@ -11,19 +11,63 @@ For Helidon 2.x releases please see [Helidon 2.x CHANGELOG.md](https://github.co
 
 For Helidon 1.x releases please see [Helidon 1.x CHANGELOG.md](https://github.com/oracle/helidon/blob/helidon-1.x/CHANGELOG.md)
 
-## [4.0.0]
-
-* WebServer no longer falls back to the default routing for additional sockets (in Helidon SE)
-* Introduced `ServerFeature` concept, server feature can access routing builders for all sockets on WebServer
-* SecurityFeature is now a WebServer feature
-* ContextFeature is now a WebServer feature
-* ObserveFeature is now a WebServer feature
-* OpenApiFeature is now a WebServer feature
-* CorsFeature is a new WebServer feature
-* TracingFeature is now an observability feature
-* Features use common config dependency - can still pass `io.helidon.Config` instance to them, only changes in SPI
-* Metrics in SE endpoint is permitted to all, unless `metrics.permit-all` is set to `false`
-* OpenAPI in SE endpoint is permitted to all, unless `openapi.permit-all` is set to `false`
+## [4.0.0-RC2]
+
+This is the second RC build of Helidon 4.0.0 and is intended as a preview release only. Do not use this release in production. It is suitable only for experimentation. APIs are subject to change. Documentation is incomplete. And some functionality is experimental.
+
+The big news in Helidon 4.0.0 is the introduction of Helidon Nima -- a ground up webserver implementation based on JDK Project Loom virtual threads. With Helidon 4 you get the high throughput of a reactive server with the simplicity of thread-per-request style programming.
```

Comment:
Níma, yes? Not Nima?

Reply:
The guidance I use is that "Helidon Níma" is OK when used as a project name, or code name for the Helidon WebServer in 4.0.0.  So I'm OK with the above (can't speak for everybody though!).

Reply:
No worries. I was mainly commenting about the accent over the i.

Reply:
You are correct! We should be consistent and use the accent.